src/services/adapters/notification-external-adapter/notification.external.adapter.spec.ts

```
@@ -0,0 +1,92 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { defaultMockerFactory } from '@test/utils/default.mocker.factory';
+import {
+  formatDatesForCalendar,
+  generateCalendarUrls,
+  generateICS,
+} from '../../../domain/timeline/event/calendar.event.calendar-links';
+import { NotificationExternalAdapter } from './notification.external.adapter';
+
+describe('NotificationExternalAdapter', () => {
+  let service: NotificationExternalAdapter;
```

Biome (2.4.4)

[error] This variable service is unused.

(lint/correctness/noUnusedVariables)